gateway-sp-comms/src/lib.rs

```diff
@@ -30,12 +33,30 @@ pub use management_switch::SpIdentifier;
 pub use management_switch::SpType;
 pub use management_switch::SwitchConfig;
 pub use management_switch::SwitchPortConfig;
+pub use management_switch::SwitchPortDescription;
 pub use single_sp::AttachedSerialConsole;
 pub use single_sp::AttachedSerialConsoleRecv;
 pub use single_sp::AttachedSerialConsoleSend;
 pub use single_sp::SingleSp;
 pub use single_sp::SpDevice;
 pub use single_sp::SpInventory;
-pub use single_sp::DISCOVERY_MULTICAST_ADDR;
 pub use timeout::Elapsed;
 pub use timeout::Timeout;
+
+const DISCOVERY_MULTICAST_ADDR: Ipv6Addr =
+    Ipv6Addr::new(0xff02, 0, 0, 0, 0, 0, 0, 1);
+const SP_MGS_PORT: u16 = 11111;
```

Comment:
Nit: I find this naming slightly confusing. Based on the comment in `default_listen_addr()` it's the listen port for the SP. The MGS part throws me off though. Since we are talking in the context of MGS in this file anyway, maybe just call it `SP_PORT`.